Tests/Integration/M365DSCTestEngine.psm1

```diff
@@ -97,6 +97,13 @@ function New-M365DSCIntegrationTest
     {
         Master -ConfigurationData $ConfigurationData -Credscredential $Credential
         Start-DscConfiguration Master -Wait -Force -Verbose -ErrorAction Stop
+
+        $TestDscConfigurationResult = Test-DscConfiguration Master -Verbose
```

Comment:
This is already handled by the workflow. Have a look in the YAML file, we are doing a Test-DSCConfiguration for every integration test already.

Reply:
Great, thanks I didn't see that. Give me a little bit, I'll see if I can implement the same checks there...